en/01_Agenda/v1/01_Installation.md

```diff
@@ -6,7 +6,7 @@ Also, some [snippets](04_Snippets/index) in the element category named
 _Agenda_ are created.
 
 There are some system settings available in the namespace _agenda_. These can
-also be edited in the [settings](02_Custom_Manager_Page/07_Settings) (cog
+also be edited in the [settings](02_Custom_Manager_Page/08_Settings) (cog
```

Comment:
FWIW, you can leave off the number when linking to avoid having to make changes when the order changes. I.e. `Custom_Manager_Page/Settings`